package.json

```diff
@@ -55,19 +55,19 @@
     "@cesium/widgets": "2.3.0"
   },
   "devDependencies": {
-    "@aws-sdk/client-s3": "^3.276.0",
-    "@aws-sdk/lib-storage": "^3.276.0",
-    "@rollup/plugin-commonjs": "^24.0.1",
-    "@rollup/plugin-node-resolve": "^15.0.1",
```

Comment:
Since the old version is still compatible, we usually don't bump the patch versions. The exception would be if the old version is no longer working for whatever reason.

Reply:
I just reverted the changes that only affected patch versions. All changes should now be minor or major version updates.